packages/integration-tests/index.js

```diff
@@ -329,6 +329,36 @@ const loadProvider = async (params = {}) =>
   await readJsonFile(params.filename)
     .then((provider) => buildProvider({ ...params, provider }));
 
+/**
+ *  Returns true if provider exists, false otherwise
+ * @param {string} stackName
+ * @param {string} id
+ * @returns {boolean}
+ */
+const providerExists = async (stackName, id) => {
+  let response;
+  const exists = await pRetry(
+    async () => {
+      try {
+        response = await providersApi.getProvider({
```

Comment:
Same comment here re: the pretry that's built in to the api client

Reply:
[`bdf616f` (#2533)](https://github.com/nasa/cumulus/pull/2533/commits/bdf616fa6e7ab7056a80c11244041f74f9424b2b)